backend/chainlit/input_widget.py

```diff
@@ -7,20 +7,19 @@
 
 
 @dataclass
-class InputWidget(ABC):
+class InputWidget:
```

Comment:
Is ABC not required when you use @abstractmethod?

https://docs.python.org/3/library/abc.html#abc.abstractmethod

Reply:
It does not break but maybe I should put it back?

Reply:
I think that you are now able to instantiate InputWidget, while otherwise you would get an exception that it can't be instantiated since it's abstract.